src/js/boardView.js

```diff
@@ -0,0 +1,70 @@
+let boardSize;
```

Comment:
Why "boardView.js" and not "BoardView.js"?

JS file names should have standards, e.g. PascalCase

Also why this file doesn't expose a class? Taking the OOP approach, only the main JS script doesn't have a class